lib/marisa/grimoire/vector/vector.h

```diff
@@ -231,8 +235,19 @@ class Vector {
     MARISA_DEBUG_IF(new_buf.get() == NULL, MARISA_MEMORY_ERROR);
     T *new_objs = reinterpret_cast<T *>(new_buf.get());
 
-    for (std::size_t i = 0; i < size_; ++i) {
-      new (&new_objs[i]) T(objs_[i]);
+#if __cplusplus >= 201103L
```

Comment:
Can this just require C++11? 

It looks `is_trivially_copyable` has been implemented by `libstdc++` for 6 years and `libc++` for a similar amount of time.

https://gcc.gnu.org/bugzilla/show_bug.cgi?id=60132